File: GymnasticsStudioServer/DTO/SubscreptionOfPaymentDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DTO
{
    public class SubscreptionOfPaymentDTO
    {
        public int PaymentId { get; set; }
        public double Sum { get; set; }
        public string StartDate { get; set; }
        public List<StudentInSubscreptionDescriptionDTO> StudentInSubscreptionDescriptionList { get; set; }



    }
}

File: GymnasticsStudioClient/src/app/classes/subscreption-of-payment-dto.ts
import { StudentInSubscreptionDescriptionDTO } from "./student-in-subscreption-description-dto"

export class SubscreptionOfPaymentDTO {

    public  PaymentId:number
    public  Sum :number
    public  StartDate :string
    public  StudentInSubscreptionDescriptionList:Array<StudentInSubscreptionDescriptionDTO>

}
